### DISCUSSION TITLE
Can new crate categories be added to the highlight tile tool please? Overgrown wooden crate, driftwood wooden crate, charred crate, metropolis crate and temple crate.

### DISCUSSION THREAD

**Discussion by iamthespork** [Ceddini/CoreKeeperMapTool#32] [Feature Requests]
Hi Ceddini! Can the crates listed below please be added as "tile highlight" options?  

**- Overgrown wooden crate:** 
- Internal ID 5636 (0), Internal name NatureWoodenDestructible

**- Driftwood wooden crate:** 
- Internal ID 5842 (0), Internal name: SeaWoodenDestructible

**- Charred crate**
- Internal ID 5863 (0), Internal name LavaWoodenDestructible

**- metropolis crate** 
- Internal ID 5840 (0), Internal Name CityDestructible

**- temple crate**
- Internal ID 5864 (0), Internal name DesertTempleDestructible

**- Large Temple Crate**
- Internal ID 5865 (0), Internal name LargeDesertTempleDestructible

I've added the Internal ID and Internal name, I'm not sure if that's helpful or not but I've added them just in case it is helpful :)

They each have different drops. I understand that they all function as wooden crates but I think adding the different types separately in the options for tile highlighting would be great. Especially since they are biome specific. 

I'm not sure if the crates listed above are being highlighted on the map when selecting the "wooden crate" option, or if that option only highlights the "wooden crate" I don't think that they are, but I'm not sure. 

Anyway I thought it would be really great to add these as their own options since they have each have different loot drops. 

Thanks for the work you do Ceddini (and team), the map tool you made is great.

**Comment by Ceddini** [comment]
If all of them have minimap colors, then I can add them. 
I'll look at it tomorrow and if they do, I'll add them right away :)

I'll also separate them into their own category if they do, that way you can easily highlight all of them at once

**Comment by iamthespork** [comment]
Awesome!!!! :) Looking at the wiki they have map colours but I do not know if they are minimap colours. :)

**Comment by Ceddini** [comment]
Tomorrow became a month later thanks to me working on my thesis. Apologies for that :) But it's added now!
